core/server/index.js

```diff
@@ -72,7 +72,11 @@ function init() {
         parentApp = require('./app')();
 
         debug('Express Apps done');
-
+    }).then(function () {
+        return auth.validation.switch({
```

Comment:
I will clean up the `passport` file to achieve this.
Wanted todo this anyway and have it almost done on a different branch.